frontend/models/product-list/index.ts

```diff
@@ -47,16 +47,17 @@ export type {
 export async function findProductList(
    filters: ProductListFiltersInput
 ): Promise<ProductList | null> {
-   const result = await strapi.getProductList({
-      filters,
-   });
+   const deviceTitle = filters.deviceTitle?.eq ?? '';
```

Comment:
This seems like a change in behavior. We never want to use `productList.deviceTitle` anymore?

Reply:
Apparently the strapi api is case-sensitive so I'm not worried about the two being different. I don't think they'll be a behavior change